netbout-client/src/test/java/com/netbout/client/RtAttachmentsITCase.java

```diff
@@ -68,16 +69,15 @@ public void postsAndReads() throws Exception {
         final String name = "test";
         attachments.create(name);
         final Attachment attachment = attachments.get(name);
+        final byte[] data = RandomUtils.nextBytes(100);
```

Comment:
@dmzaytsev this introduces a non-deterministic component into the test. I believe it would be better to have one fixed binary array instead of using `RandomUtils`.